cdp/cdp_parser.py

```diff
@@ -255,27 +255,27 @@ def load_default_args(self, files=[]):
             '-p', '--parameters',
             type=str,
             dest='parameters',
-            help='Path to the user-defined parameter file',
+            help='Path to the user-defined parameter file.',
             required=False)
         self.add_argument(
             '-d', '--diags',
             type=str,
             nargs='+',
             dest='other_parameters',
             default=[],
-            help='Path to the other user-defined parameter file',
+            help='Path to the other user-defined parameter file.',
             required=False)
         self.add_argument(
             '-n', '--num_workers',
             type=int,
             dest='num_workers',
-            help='Number of workers, used when running with multiprocessing or distributedly',
+            help='Number of workers, used when running with multiprocessing or distributedly.',
```

Comment:
@zshaheen as far as my english stretches, `distributedly` isn't a word. Might I suggest taking the edit below "Used when running in distributed mode" and also applying it here :)

Reply:
@zshaheen it seems I'm wrong (https://www.merriam-webster.com/dictionary/distributedly) but I still think my text is clearer

Reply:
Okay, done in 1a7cfc3dc4899bfd3c43c5f72e9484912535aced